Api edits (#358)
cmu-delphi/delphi-epidata: docs/api/covidcast-signals/safegraph.md

* **First issued:** June 20, 2020
* **Number of data revisions since June 23, 2020:** 1
* **Date of last change:** [November 3, 2020](../covidcast_changelog.html#safegraph)

> Delphi's data source is based on [social distancing metrics](https://docs.safegraph.com/docs/social-distancing-metrics).  SafeGraph

Reviewer: ```suggestion
This data source is based on [SafeGraph's social distancing metrics](https://docs.safegraph.com/docs/social-distancing-metrics).  SafeGraph
```

provides this data for individual census block groups, using differential
privacy to protect individual people's data privacy.

> Delphi creates the SafeGraph data features at the census block group level, then aggregates these features to the county and state levels. The aggregated data is freely available through the COVIDcast API.

Reviewer: ```suggestion
Delphi uses this information to create features at the census block group level, then aggregates these features to the county and state levels. The aggregated data is freely available through the COVIDcast API.
```

* **Number of data revisions since June 23, 2020:** 0
* **Date of last change:** never

> Delphi's data source is based on SafeGraph's [Weekly Patterns](https://docs.safegraph.com/docs/weekly-patterns) dataset. SafeGraph provides this data for different points of interest ([POIs](https://docs.safegraph.com/v4.0/docs#section-core-places)) considering individual census block groups, using differential privacy to protect individual people's data privacy.

Reviewer: ```suggestion
This data source is based on [SafeGraph's Weekly Patterns](https://docs.safegraph.com/docs/weekly-patterns) dataset. SafeGraph provides this data for different points of interest ([POIs](https://docs.safegraph.com/v4.0/docs#section-core-places)) considering individual census block groups, using differential privacy to protect individual people's data privacy.
```

Delphi gathers the number of daily visits to POIs of certain types(bars,

> restaurants, etc.)  from SafeGraph's Weekly Patterns data at the 5-digit ZipCode level aggregates and reports these features to the county, MSA, HRR, and state levels. The aggregated data is freely available through the COVIDcast API.

Reviewer: ```suggestion
restaurants, etc.)  from SafeGraph's Weekly Patterns data at the 5-digit ZipCode level, then aggregates these features to the county, MSA, HRR, and state levels. The aggregated data is freely available through the COVIDcast API.
```

| `restaurants_visit_num` | The number of daily visits made by those with SafeGraph's apps to restaurant-related POIs in a certain region <br/> **First Available:** 01/01/2019 |
| `restaurants_visit_prop` | The number of daily visits made by those with SafeGraph's apps to restaurant-related POIs in a certain region, per 100,000 population <br/> **First Available:** 01/01/2019 |

> SafeGraph delivers the number of daily visits to U.S. POIs, the details described in the [Places

Reviewer: ```suggestion
SafeGraph delivers the number of daily visits to U.S. POIs described in the [Places
```
